visualization/app/codeCharta/state/effects/updateVisibleTopLabels/updateVisibleTopLabels.effect.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { Store as PlainStoreUsedByEffects, Store } from "../../store/store"
```

Comment:
I think you want to `import { Store } from "../../angular-redux/store"` here instead.  Otherwise the test does use the "real" Store, as you provide a different token in [line 40](https://github.com/MaibornWolff/codecharta/pull/3046/files#diff-52a155c6f1d868bce33384d4a1453e51ce9ffef1426de75b40eae39903191daaR40)

Reply:
Unfortunately it doesn't solve my problem :(